examples/ec2/vpcendpoint.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: ec2.aws.crossplane.io/v1alpha1
+kind: VPCEndpoint
+metadata:
+  name: sample-vpcendpoint
+spec:
+  forProvider:
+    region: us-east-1
+    privateDNSEnabled: true
+    serviceName: com.amazonaws.us-east-1.vpce-svc-0123456789abcdef0
```

Comment:
is it possible to provide values here (serviceName, securityGroupIDs, subnetIDs, vpcID) that will work with the existing examples?

Reply:
This has been added.